### ISSUE TITLE
Logs error  for class

### ISSUE THREAD

**Issue by sagitt** [tomaae/homeassistant-openmediavault#68]
## Describe the issue
I see this in logs:

```
Logger: homeassistant.helpers.entity
Source: helpers/entity.py:580
First occurred: 23:02:58 (18 occurrences)
Last logged: 23:03:03

Entity sensor.nas_sda (<class 'custom_components.openmediavault.sensor.OMVDiskSensor'>) implements device_state_attributes. Please report it to the custom component author.
Entity sensor.nas_sdb (<class 'custom_components.openmediavault.sensor.OMVDiskSensor'>) implements device_state_attributes. Please report it to the custom component author.
Entity binary_sensor.nas_update_available (<class 'custom_components.openmediavault.binary_sensor.OMVBinarySensor'>) implements device_state_attributes. Please report it to the custom component author.
Entity binary_sensor.nas_reboot_pending (<class 'custom_components.openmediavault.binary_sensor.OMVBinarySensor'>) implements device_state_attributes. Please report it to the custom component author.
Entity binary_sensor.nas_config_dirty (<class 'custom_components.openmediavault.binary_sensor.OMVBinarySensor'>) implements device_state_attributes. Please report it to the custom component author.
```

## Software versions

 - Home Assistant version: 2022.2.5
 - openmediavault integration version: 1.1.7
 - openmediavault software version: 5.2.65-1

**Comment by tomaae** [comment]
Wasnt that already fixed? #65

**Comment by sagitt** [comment]
> Wasnt that already fixed? #65 

I can see the error in my logs.

Maybe fixed here but probably never released an upgrade

**Comment by tomaae** [comment]
no, new version was not released. fix is available in master only right now
